packages/synapse-react-client/src/components/ChangePassword/ChangePassword.integration.test.tsx

```diff
@@ -520,6 +520,9 @@ describe('ChangePassword tests', () => {
 
     await user.click(submitButton)
 
+    // Wait for the UI to update before we change the server behavior
+    await getTOTPInputs()
+
```

Comment:
Fixes flaky test. This test makes two calls to the same service. With this change we now wait for the UI to update after making the first call (that has a 2FA challenge) before preparing the mock server with the second mock response. Before, I think sometimes the server behavior would update before the first API call was even made.